src/Game/Managers/CommandManager.cs

```diff
@@ -87,5 +94,12 @@ public static void Execute(string name, params string[] args)
             else
                 Log.Message(LogTypes.Warning, $"Commad: '{name}' not exists");
         }
+
+        public static void OnHueTarget(Entity entity)
+        {
+            TargetManager.TargetGameObject(entity);
+            Mouse.LastLeftButtonClickTime = 0;
+            GameActions.Print($"Item ID: {entity.Graphic}\nHue: {entity.Hue}");
+        }
```

Comment:
:bulb: This is a pattern that didn't exist before. It didn't seem right to have the `-hue` command logic spread across 4-5 different files like `-info` is currently. In RunUO, each command is its own class altogether and is registered in a static `#Initialize()` method. That may be a pattern to follow here, too.